Yank/multistate/replicaexchange.py

```diff
@@ -209,6 +209,9 @@ def _repex_mixing_scheme_validator(instance, replica_mixing_scheme):
             if replica_mixing_scheme not in supported_schemes:
                 raise ValueError("Unknown replica mixing scheme '{}'. Supported values "
                                  "are {}.".format(replica_mixing_scheme, supported_schemes))
+            if instance.locality != None:
```

Comment:
Small pep8 thing, you should be able to do `if ... and ....` since its back to back `if`s. Assuming you dont plan to do anything else between the two `if`s here

Reply:
I don't understand.

Reply:
`if instance.locality != None and replica_mixing_scheme not in ['swap-neighbors']` instead of `if` on one line, and a second `if` on another